docs/source/build.md

```diff
@@ -1,318 +1,184 @@
+[Conda](https://docs.anaconda.com/free/miniconda/index.html) environment scripts are provided for installing the necessary dependencies to build cuVS from source. It is preferred to use `mamba`, as it provides significant speedup over `conda`:
 ```bash
-mamba env create --name rapids_raft -f conda/environments/all_cuda-120_arch-x86_64.yaml
-mamba activate rapids_raft
+mamba env create --name cuvs -f conda/environments/all_cuda-120_arch-x86_64.yaml
+mamba activate cuvs
 ```
 
-All of RAFT's C++ APIs can be used header-only and optional pre-compiled shared libraries provide some host-accessible runtime APIs and template instantiations to accelerate compile times.
-
 The process for building from source with CUDA 11 differs slightly in that your host system will also need to have CUDA toolkit installed which is greater than, or equal to, the version you install into you conda environment. Installing CUDA toolkit into your host system is necessary because `nvcc` is not provided with Conda's cudatoolkit dependencies for CUDA 11. The following example will install create and install dependencies for a CUDA 11.8 conda environment
 ```bash
-mamba env create --name rapids_raft -f conda/environments/all_cuda-118_arch-x86_64.yaml
-mamba activate rapids_raft
+mamba env create --name cuvs -f conda/environments/all_cuda-118_arch-x86_64.yaml
+mamba activate cuvs
 ```
 
-The recommended way to build and install RAFT from source is to use the `build.sh` script in the root of the repository. This script can build both the C++ and Python artifacts and provides CMake options for building and installing the headers, tests, benchmarks, and the pre-compiled shared library.
-
-### Header-only C++
-
-`build.sh` uses [rapids-cmake](https://github.com/rapidsai/rapids-cmake), which will automatically download any dependencies which are not already installed. It's important to note that while all the headers will be installed and available, some parts of the RAFT API depend on libraries like CUTLASS, which will need to be explicitly enabled in `build.sh`.
-
-The following example will download the needed dependencies and install the RAFT headers into `$INSTALL_PREFIX/include/raft`. 
-```bash
-./build.sh libraft
-```
-The `-n` flag can be passed to just have the build download the needed dependencies. Since RAFT's C++ headers are primarily used during build-time in downstream projects, the dependencies will never be installed by the RAFT build.
-```bash
-./build.sh libraft -n
-```
+The recommended way to build and install cuVS from source is to use the `build.sh` script in the root of the repository. This script can build both the C++ and Python artifacts and provides CMake options for building and installing the headers, tests, benchmarks, and the pre-compiled shared library.
```

Comment:
More of a curiosity question probably for @benfred than anything else, can we add the rust build also to build.sh? Should be possible since it's just doing the `cargo` command, no?

Reply:
I'll be adding the rust build to build.sh in a future PR - we can definitely add this in

Reply:
I was talking w/ Bed about that earlier. Yes, the plan is ultimately to add that to build.sh